teachopencadd/talktorials/019_One_Hot_Encoding/talktorial.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>I suggest to make the function name more easy to understand, like <code>one_hot_matrix</code> ,<code>one_hot_matrix_plot.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Done :)</p><p> </p>